ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -5,14 +5,14 @@ package com.daml.http.dbbackend
 
 import com.daml.lf.data.Ref
 import com.daml.nonempty
-import nonempty.{NonEmpty, +-:}
+import nonempty.{+-:, NonEmpty}
```

Comment:
If possible, I would try not to intervene in the import sequence adopted by other teams. I am guessing the formatter did this automatically?

Reply:
Yes. I will try to remove it.